tests/test_publish.py

```diff
@@ -31,6 +32,13 @@ def scenario_dist() -> Generator[Path, None, None]:
         yield dist
 
 
+@pytest.fixture
+def credentials(tmpenviron) -> None:
+    os.environ["PACKSE_PUBLISH_USERNAME"] = "username"
+    os.environ["PACKSE_PUBLISH_PASSWORD"] = "password"
+    yield
```

Comment:
I'm kinda scared of having process globals in fixtures, but tbh i don't know enough details about pytest

Reply:
I've got a temporary environment fixture that ensures the changes are rolled back. The tests should never be run in multiple threads so it should be safe.